plot-api/src/jsMain/kotlin/jetbrains/letsPlot/defaultFrontendContext.kt

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2022. JetBrains s.r.o.
+ * Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+ */
+
+package jetbrains.letsPlot
+
+actual fun createDefaultFrontendContext(): FrontendContext {
+    TODO("Not yet implemented")
```

Comment:
Return an instance throwing a IllegalStateException - similar to JVM variant.

Reply:
Done